src/pilet-validate.test.ts

```diff
@@ -18,4 +18,89 @@ runTests('pilet-validate', ({ test, setup }) => {
 
     expect(result).toContain('Validation succeeded with 1 warning(s)');
   });
+
+  test('outdated-appShell-version', 'outdated appShell version should yield warning', [], async (ctx) => {
+    let appShellName;
+    await ctx.assertFiles({
+      async 'package.json'(content: string) {
+        const appShell = await JSON.parse(content);
+        appShellName = appShell.name;
+      },
+    });
+
+    await ctx.setFiles({
+      'node_modules/sample-piral/package.json'(content: string) {
+        const packageJson = JSON.parse(content);
+        packageJson.name = appShellName;
+        return JSON.stringify(packageJson, undefined, 2);
+      },
+    });
+
+    const result = await ctx.run(`npx pilet validate`);
+
+    expect(result).toContain(`The used version of \"${appShellName}\" is outdated.`);
+    expect(result).toContain(`Validation succeeded with 1 warning(s).`);
+  });
+
+  test('bundle-size', 'when the bundle size is over the limit should error out', [], async (ctx) => {
```

Comment:
I just see the error code - I'd also like to have the warning case (and a case where we disable this with `0`).